core/src/main/java/org/apache/iceberg/view/SQLViewRepresentationParser.java

```diff
@@ -35,7 +35,7 @@ class SQLViewRepresentationParser {
   private static final String DEFAULT_CATALOG = "default-catalog";
   private static final String DEFAULT_NAMESPACE = "default-namespace";
   private static final String FIELD_ALIASES = "field-aliases";
-  private static final String FIELD_COMMENTS = "field-comments";
+  private static final String FIELD_DOCS = "field-docs";
```

Comment:
Replied [here](https://github.com/apache/iceberg/pull/7416#discussion_r1175521596) as well, I think we should change the spec to be field-comments instead because most mechanisms for creating/altering the view (such as SQL) will call this a comment. Let me know your thoughts on this though

Reply:
I guess `docs` would have some consistency with `NestedField` in `Schema` that has a `doc` field. When using SQL we also refer to that as comment, e.g. in Hive

```
ALTER TABLE test_change CHANGE a1 a1 INT COMMENT 'this is column a1';
```

So I am not too concerned with the naming choice here. 

But even if we go with `docs`, should we use `doc` to be more consistent?

Reply:
I updated to comments. I think we can still make a few changes like this where needed and I agree that comments works better.

The original logic was to be like "doc" in the schema fields, but that's already there and this is a separate concept that has multiple fields. I don't think we're losing much by making it more similar to the SQL.